langchain/chains/conversation/base.py

```diff
@@ -0,0 +1,54 @@
+"""Chain that carries on a conversation and calls an LLM."""
+from typing import Any, Dict, List
+
+from pydantic import BaseModel, Extra
+
+from langchain.chains.memory import MemoryChain
+from langchain.docstore.in_memory import Docstore
+from langchain.llms.base import LLM
+
+
+class ConversationChain(MemoryChain, BaseModel):
+    """Chain to have a conversation and load context from memory.
+
+    Example:
+        .. code-block:: python
+
+            from langchain import ConversationChain, InMemoryDocstore, OpenAI
+            conversation = ConversationChain(llm=OpenAI(), docstore=InMemoryDocstore())
+    """
+
+    llm: LLM
```

Comment:
for the ones that are defind in Memory chain, i dont think you need to redefine them